### ISSUE TITLE
Integration Test cases are failing for Power(ppc64le).

### ISSUE THREAD

**Issue by Sunidhi-Gaonkar1** [aquasecurity/trivy#2612]
Hi Team, I am raising this issue in reference to #2570 . Unit test cases and Module Integration test cases are passing after the fix done in #2575 .Currently facing issues for integration tests.
The images `cesanta/docker_auth:1` , `registry:2.7.0 ` , `ghcr.io/aquasecurity/trivy-test-images/containerd:latest` are not supported on ppc64le. After using higher version of `registry image (2.8.0)`, as it has ppc64le support, and building `ghcr.io/aquasecurity/trivy-test-images/containerd:latest` and `cesanta/docker_auth:1` for ppc64le , the integration tests are still failing.
The test `trivy/pkg/fanal/test/integration/containerd_test.go` is architecture specific (amd64). 
I have attached the logs below for the reference. Pointers regarding the same will be helpful. Thank you.
Log : [Integration_test_logs.txt](https://github.com/aquasecurity/trivy/files/9206336/Integration_test_logs.txt)

**Comment by Sunidhi-Gaonkar1** [comment]
Hello @knqyf263 , are all tests being run only via GitHub Actions? or is there any internal testing CI happening on x86? At this moment GitHub Actions doesn't support native ppc64le builds for CI. The capability is present on Travis-CI, would like to know if the community is interested in enabling it for testing on Power.

**Comment by afdesk** [comment]
Hello @Sunidhi-Gaonkar1! I'm trying to fix integration tests.
is there a simple way to run them somewhere on ppc64le?

**Comment by Sunidhi-Gaonkar1** [comment]
Yes, would like to share further details on slack, I have posted a message on slack regarding the same.

**Comment by knqyf263** [comment]
> Hello @knqyf263 , are all tests being run only via GitHub Actions? or is there any internal testing CI happening on x86?

We use GitHub Actions only.

>The capability is present on Travis-CI, would like to know if the community is interested in enabling it for testing on Power.

AFAIK, Travis CI is not free for us.

>Project must not be sponsored by a commercial company or organization (monetary or with employees paid to work on the project)

https://daniel.haxx.se/blog/2021/06/14/bye-bye-travis-ci/

**Comment by afdesk** [comment]
just FYI.
I've tested `Trivy` on demo pcc64le instance.
it seems that `trivy_0.31.2_Linux-PPC64LE.tar.gz` works correctly.
but i could't build the code on this machine yet, there is no enough disk space...
asked them to increase hdd )

**Comment by Sunidhi-Gaonkar1** [comment]
> AFAIK, Travis CI is not free for us.

@knqyf263 For Travis,Power builds should be covered under OSS and you can apply for extra credits / points via travis support ticket.
